server/server.go

```diff
@@ -442,6 +465,19 @@ func (server *Server) finaliseConn(ctx context.Context, conn session.Conn, l Lis
 		server.log.Debugf("connection %v failed spawning: %v\n", conn.RemoteAddr(), err)
 		return
 	}
+
+	itemComponentEntries := make([]protocol.ItemComponentEntry, len(server.itemComponents))
+	for name, component := range server.itemComponents {
+		itemComponentEntries = append(itemComponentEntries, protocol.ItemComponentEntry{
+			Name: name,
+			Data: component,
+		})
+	}
+	if err := conn.WritePacket(&packet.ItemComponent{Items: itemComponentEntries}); err != nil {
+		_ = l.Disconnect(conn, "Internal server error.")
```

Comment:
Just ignore what's returned instead of pulling a pocketmine and having a random iNterNal SerVer erRoR